tabular/src/autogluon/tabular/trainer/auto_trainer.py

```diff
@@ -15,10 +15,11 @@ def get_models(self, hyperparameters, hyperparameter_tune=False, level='default'
                                  num_classes=self.num_classes, hyperparameters=hyperparameters, hyperparameter_tune=hyperparameter_tune, level=level, extra_ag_args_fit=extra_ag_args_fit)
 
     # TODO: rename to .fit for 0.1
-    def train(self, X_train, y_train, X_val=None, y_val=None, hyperparameter_tune=False, feature_prune=False, holdout_frac=0.1, hyperparameters=None, ag_args_fit=None, excluded_model_types=None, time_limit=None, **kwargs):
+    def train(self, X_train, y_train, X_val=None, y_val=None, X_unlabeled=None, hyperparameter_tune=True, feature_prune=False, holdout_frac=0.1, hyperparameters=None, ag_args_fit=None, excluded_model_types=None, unlabeled_data=None, time_limit=None, **kwargs):
```

Comment:
bug: Don't change default values (hyperparameter_tune=False)

Why X_unlabeled and unlabeled_data?

Reply:
Changed hyperparameter_tune = True. Seems like this was an oversight, removed unlabeled_data param. It wasn't being used.